jupyter-js-widgets/src/embed-webpack.js

```diff
@@ -7,13 +7,12 @@
 // Load jquery and jquery-ui
 var $ = require('jquery');
 require('jquery-ui');
-$.getQueryParameters = window.$.getQueryParameters;
```

Comment:
These lines are left over from the docs, without these lines the docs will break.  However, I don't think these lines should exist here, instead the fix belongs in the docs.